addons/website/static/src/js/menu/content.js

```diff
@@ -339,6 +345,26 @@ var PagePropertiesDialog = weWidgets.Dialog.extend({
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * @private
+     */
+    _onMetaDescriptionChanged: function (ev) {
+        var $textarea = $(ev.target);
```

Comment:
Can you retrieve the textarea without using ev (website_meta_description)? 
And then call _onMetaDescriptionChanged on start event of the widget ?
That way the warning will be shown if there is a not required size description when loading the widget

Reply:
It doesn't work in this case, for some reason the widget is not yet displayed in the start method here?

Reply:
Ok fixed